INFO     log:logger_util.py:61 
-------------接口请求开始-----------
INFO     log:logger_util.py:61 接口名称：查成分(清数据后)
INFO     log:logger_util.py:61 接口方式：get
INFO     log:logger_util.py:61 接口路径：http://127.0.0.1:18000/api/v1/runtimeformula/1/ingredient/get
INFO     log:logger_util.py:61 请求头：{'Authorization': 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJodHRwOi8vc2NoZW1hcy54bWxzb2FwLm9yZy93cy8yMDA1LzA1L2lkZW50aXR5L2NsYWltcy9uYW1lIjoiU3VwZXJBZG1pbiIsImh0dHA6Ly9zY2hlbWFzLnhtbHNvYXAub3JnL3dzLzIwMDUvMDUvaWRlbnRpdHkvY2xhaW1zL25hbWVpZGVudGlmaWVyIjoiU3VwZXJBZG1pbiIsImlzcyI6IkNNU3YyIiwiYXVkIjoiQXBpIn0.AJUOE6YQ5MAxDqHOIz-yQbpCKPNq5tPoFBWUY381qns', 'X-Project': 'a591d8d3add24b98a9e20d034e0ffbfc'}
INFO     log:logger_util.py:61 文件上传：None
INFO     log:logger_util.py:61 预期结果：[{'equals': {'status_code': 500}}, {'not_contain': 'name'}]
INFO     log:logger_util.py:61 实际结果：{'exception': {'when': '2024-07-24T13:28:15.8611646+08:00', 'message': 'Object reference not set to an instance of an object.', 'type': 'NullReferenceException', 'callStack': [{'assembly': 'System.Private.CoreLib', 'fileName': 'C:\\GitLab-Runner\\builds\\uF5yEnPSn\\0\\cms\\cms\\src\\Units\\Formula\\CMS.Unit.Formula\\Services\\RuntimeFormulaService.cs', 'method': 'MoveNext', 'lineNumber': 283}, {'assembly': 'System.Private.CoreLib', 'fileName': 'C:\\GitLab-Runner\\builds\\uF5yEnPSn\\0\\cms\\cms\\src\\Plugins\\CMS.Plugin.Formula\\Controller\\RuntimeFormulaController.cs', 'method': 'MoveNext', 'lineNumber': 144}, {'assembly': 'System.Private.CoreLib', 'fileName': 'C:\\GitLab-Runner\\builds\\uF5yEnPSn\\0\\cms\\cms\\src\\Framework\\CMS.Framework.Authorization.AspNetCore\\Middlewares\\AuthorizationMiddleware.cs', 'method': 'MoveNext', 'lineNumber': 47}, {'assembly': 'System.Private.CoreLib', 'fileName': 'C:\\GitLab-Runner\\builds\\uF5yEnPSn\\0\\cms\\cms\\src\\CMS.Core\\TraceMiddleware.cs', 'method': 'MoveNext', 'lineNumber': 21}, {'assembly': 'System.Private.CoreLib', 'fileName': 'C:\\GitLab-Runner\\builds\\uF5yEnPSn\\0\\cms\\cms\\src\\Framework\\CMS.Framework.AspNetCore\\Middlewares\\ExceptionMonitorMiddleware.cs', 'method': 'MoveNext', 'lineNumber': 36}], 'innerException': None}, 'code': 500, 'msg': 'Object reference not set to an instance of an object.'}
INFO     log:logger_util.py:61 接口请求成功！
INFO     log:logger_util.py:61 
-------------接口请求结束-----------